Indico/Query/ModelGroupQuery.cs

```diff
@@ -40,20 +31,9 @@ public ModelGroupQuery(GraphQLHttpClient graphQLHttpClient)
         /// </summary>
         /// <returns>ModelGroupQuery</returns>
         /// <param name="id">Identifier.</param>
-        public ModelGroupQuery SetId(int id)
-        {
-            this.Id = id;
-            return this;
-        }
-
-        /// <summary>
-        /// Use to query ModelGroup by name
-        /// </summary>
-        /// <returns>ModelGroupQuery</returns>
-        /// <param name="name">Name.</param>
-        public ModelGroupQuery Name(string name)
+        public ModelGroupQuery SetId(int mgId)
```

Comment:
Instead of providing a new setter, we can use the existing one. we can also add docs to public variables directly.
For instance, Here this can be replaced to
/// <summary>
/// Used to query ModelGroup by id
/// </summary>
/// <value>Identifier.</value>
public int MgId { get; set; }